R/manage.R

```diff
@@ -25,12 +25,30 @@
 #'   "known-good-versions" list from the Google Chrome for Testing versions at
 #'   <https://googlechromelabs.github.io/chrome-for-testing>.
 #'
-#' @param version A character string specifying the version to use. Default is
-#'   `"latest-installed"`, which uses the latest version you have installed via
-#'   chromote. To use the current latest version, downloading the binary if
-#'   necessary. If you specify a partial version, e.g. `"132"`, chromote will
-#'   find the most recent release matching that version. Use
-#'   `version = "system"` to revert back to using the system-installed Chrome.
+#' @param version A character string specifying the version to use. The default
+#'   value is `"latest-stable"` to follow the latest stable release of Chrome.
+#'   For robust results, and to avoid frequently downloading new versions of
+#'   Chrome, use a fully qualified version number, e.g. `"133.0.6943.141"`.
+#'
+#'   If you specify a partial version, e.g. `"133"`, chromote will find the most
+#'   recent release matching that version, preferring to use the latest
+#'   *installed* release that matches the partially-specified version. chromote
+#'   also supports a few special version names:
+#'
+#'   * `"latest-installed"`: The latest version currently installed locally in
+#'     chromote's cache. If you don't have any installed versions of the binary,
+#'     chromote uses `"latest"`.
```

Comment:
`"latest-installed"` feels like it should fall back to `"latest-stable"`.

Reply:
Yeah I agree, but it's currently a recursive call and it'd take a little work to untangle. I think having the default be `"latest-stable"` is going to mitigate the downsides and makes that work less important. If you've run `with_chrome_version()` even _once_ before changing the version to `"latest-installed"`, you won't run into this.

Reply:
🥳